core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1200,7 +1204,7 @@ public Void visitValues(ValuesNode node, Context context)
                     })
                     .collect(toImmutableList());
             for (String row : rows) {
-                nodeOutput.appendDetails(row);
+                nodeOutput.appendDetails("%s", row);
```

Comment:
Do we need to add here formatting?

Reply:
Yes, because otherwise `rows` would be interpreted as a format string. Error Prone cannot prove that it does not contain any format metacharacters which would be misinterpreted.